Makefile

```diff
@@ -227,10 +227,11 @@ endif
 get_coverage:
 	@echo "Copying coverage report from build image to genhtml if exist..."
 	@docker create -ti --name $(OVMS_CPP_CONTAINTER_NAME) $(OVMS_CPP_DOCKER_IMAGE)-build:$(OVMS_CPP_IMAGE_TAG) bash
-	@docker cp $(OVMS_CPP_CONTAINTER_NAME):/ovms/genhtml/ . 2>/dev/null
+	#@docker cp $(OVMS_CPP_CONTAINTER_NAME):/ovms/genhtml/ . 2>/dev/null
```

Comment:
while copying the report is skipped, it will never be checked

Reply:
Good catch  forgot to remove.